Packs/SkyboxSecurity/Integrations/SkyboxTicketsService/SkyboxTicketsService.py

```diff
@@ -0,0 +1,2281 @@
+
+    response = client.service.findAccessChangeTickets(filter, subRange)
+
+    command_results = CommandResults(
+        outputs_prefix='Skybox.findAccessChangeTickets',
+        outputs_key_field='',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getTicketFields_command(client, args):
+    ticketId = args.get('ticketId')
+    ticketIdType = args.get('ticketIdType')
+
+    response = client.service.getTicketFields(ticketId, ticketIdType)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketFields',
+        outputs_key_field='',
+        outputs=helpers.serialize_object(response),
+        raw_response=helpers.serialize_object(response)
+    )
+
+    return command_results
+
+
+def getTicketsImplementedChangeRequests_command(client, args):
+    ticketIds = args.get('ticketIds')
+
+    response = client.service.getTicketsImplementedChangeRequests(ticketIds)
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketsImplementedChangeRequests',
+        outputs_key_field='',
+        outputs=serialize_object_list(response),
+        raw_response=serialize_object_list(response)
+    )
+
+    return command_results
+
+
+def getTicketDeferChangeRequestsCalculationStatus_command(client, args):
+    ticketId = args.get('ticketId')
+    status = {}
+
+    response = client.service.getTicketDeferChangeRequestsCalculationStatus(ticketId)
+    status['status'] = response
+    command_results = CommandResults(
+        outputs_prefix='Skybox.getTicketDeferChangeRequestsCalculationStatus',
+        outputs_key_field='',
+        outputs=status,
+        raw_response=status
+    )
+
+    return command_results
+
+
+def test_module(client):
+    response = client.service.testService(111000111)
+    if response == 111000111:
+        return_results('ok')
+    else:
+        return_results(str(response))
+
+
+def main():
+    handle_proxy()
+    params = demisto.params()
+    args = demisto.args()
+    url = params.get('url')
+    verify_certificate = not params.get('insecure', False)
+
+    wsdl = url + "/skybox/webservice/jaxws/tickets?wsdl"
+
+    username = params['credentials']['identifier']
+    password = params['credentials']['password']
+
+    session: Session = Session()
+    session.auth = (username, password)
+    session.verify = verify_certificate
+    cache: SqliteCache = SqliteCache(path=get_cache_path(), timeout=None)
+    transport: Transport = Transport(session=session, cache=cache)
+    settings: Settings = Settings(strict=False, xsd_ignore_sequence_order=True)
+
+    command = demisto.command()
+    demisto.debug(f'Command being called is {command}')
+
+    try:
+        requests.packages.urllib3.disable_warnings()
+        client = zClient(wsdl=wsdl, transport=transport, settings=settings)
+
+        commands = {
+            'skybox-getTicketWorkflow': getTicketWorkflow_command,
```

Comment:
I would prefer to leave them as is. This is because you can easily look them up in the vendor doco.

Reply:
Since the support is partner, the command names should be like our convention standards, as you can see [here](https://xsoar.pan.dev/docs/tutorials/tut-design#integration-naming-conventions).

Reply:
fixed in commit `c0d0836bb31d099c7b5dd16d4a2240e4cb2dbe9d`